torch/_inductor/codegen/cpp_micro_gemm.py

```diff
@@ -603,15 +603,12 @@ class CppMicroGemmAMX(CppMicroGemm):
 
 {%- if input_dtype == torch.bfloat16 and input2_dtype == torch.int8 %}
     // create a buffer for tiles of B.
-    // TODO: loop-unrolling of the "compute" lambda may result in incorrect output
-    // as this buffer would be used, so maybe 4 of these should be used?
-    // Since UT output is correct, looks like loop unrolling isn't actually happening.
-    alignas(64) {{input_t}} bf16_weights_buf[512];
+    // alignment of 128 is being used to prevent segfault on RHEL with some specific libstdc++ version.
+    alignas(128) {{input_t}} bf16_weights_buf[512];
```

Comment:
I suspect this is our local test environment issue. Better not "work around" it before understanding the root cause of it.